### ISSUE TITLE
SES: Floating point exception(core dumped)

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/pbdmms#151]
Just run:

```
./kewe_profile: line 4:  3412 Floating point exception(core dumped) ./kewe --profile
```

**Comment by richelbilderbeek** [comment]
Correct:

```
if (t%p.m_output_parameters.outputfreq == 0)
```

I allow `outputfreq` to be zero if the user wants output only at the end.

**Comment by richelbilderbeek** [comment]
Fixed with e457e52f97e4d6aaeb56971f5f94d6cd5d25fdda
